mewarpx/mewarpx/setups_store/diode_setup.py

```diff
@@ -39,6 +39,10 @@ class DiodeRun_V1(object):
     # ### VACUUM ###
     # Cathode anode distance
     D_CA = 1e-3
+    # Vacuum bias of the highest voltage grid relative to the cathode
+    V_GRID = 100
+    # If calculating DT using a CFL factor, this is the CFL factor
+    CFL_FACTOR = 1
```

Comment:
Should these values be none, since we're not supposed to have default values for these?